xml/System.Runtime.InteropServices.JavaScript/JSObject.xml

```diff
@@ -72,9 +72,9 @@
       </Parameters>
       <Docs>
         <param name="propertyName">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>Returns value of the property as <see cref="T:System.Boolean" /> if the property exists, otherwise <see langword="false" />.</summary>
```

Comment:
```suggestion
        <summary>Returns the value of the specified property as <see cref="T:System.Boolean" /> if the property exists, otherwise <see langword="false" />.</summary>
```